BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -126,42 +136,84 @@ public BraintreeClient(@NonNull Context context, @NonNull String authorization,
      *
      * @param callback {@link ConfigurationCallback}
      */
-    public void getConfiguration(@NonNull ConfigurationCallback callback) {
-        configurationLoader.loadConfiguration(applicationContext, authorization, callback);
+    public void getConfiguration(@NonNull final ConfigurationCallback callback) {
```

Comment:
I don't think this is a breaking change but we could capture the callback as `final` internally if needed.

Reply:
Agreed - I think this `final` here only impacts our ability to alter the callback param within this method.